engine/audio/src/audiocapture.cpp

```diff
@@ -166,22 +168,21 @@ double AudioCapture::fillBandsData(int number)
             magnitudeSum += qSqrt((((fftw_complex*)m_fftOutputBuffer)[i][0] * ((fftw_complex*)m_fftOutputBuffer)[i][0]) +
                                   (((fftw_complex*)m_fftOutputBuffer)[i][1] * ((fftw_complex*)m_fftOutputBuffer)[i][1]));
         }
-        double bandMagnitude = (magnitudeSum / subBandWidth);
+        bandMagnitude = (magnitudeSum / subBandWidth);
         m_fftMagnitudeMap[number].m_fftMagnitudeBuffer[b] = bandMagnitude;
-        if (maxMagnitude < bandMagnitude)
-            maxMagnitude = bandMagnitude;
     }
 #else
     Q_UNUSED(number)
 #endif
-    return maxMagnitude;
+    return bandMagnitude;
 }
 
 void AudioCapture::processData()
 {
 #ifdef HAS_FFTW3
     unsigned int i;
-    quint64 pwrSum = 0;
```

Comment:
I think this needs to be a floating point value, not an integer (?)

Reply:
Sum of the power is used to calculate an average, so it's a very big number. I needed a 64 bit variable. I think float is not needed at this stage since `m_audioBuffer[i]` contains `int16_t` values.

Reply:
I realised that the FFT is supposed to be on values -1.0 to 1.0 for the signal, so I rescaled the input values.